File: game-server/src/main/java/org/forfun/mmorpg/game/attribute/AttributeType.java
package org.forfun.mmorpg.game.attribute;

public enum AttributeType {
	
	Attack(1, Boolean.FALSE),
	
	Defense(1, Boolean.FALSE),
	
	Hp(1, Boolean.FALSE),
	
	
	;
	
	
	int id;
	/**
	 * 是否万分比
	 */
	boolean percent;
	
	AttributeType(int id, boolean percent) {
		this.id = id;
		this.percent = percent;
	}

}

File: game-server/src/main/kotlin/org/forfun/mmorpg/game/attribute/AttributeType.kt
package org.forfun.mmorpg.game.attribute

enum class AttributeType(
    val id: Int,
    /**
     * 是否万分比
     */
    val percent: Boolean
) {

    Attack(1, false),
    Defense(1, false),
    Hp(1, false)
}
